doc/Command_Index_internal.tex

```diff
@@ -585,10 +582,10 @@ \section{Gregorio\TeX{} Controls}
 Macro called before each glyph.
 
 \macroname{\textbackslash gre@normalinitial}{}{gregoriotex-main.tex}
-Macro called at the end of the score to ensure that a big initial setting doesn't carry into the next score.
+Macro called at the end of the score to ensure that a big initial setting doesn’t carry into the next score.
 
 \macroname{\textbackslash greoldcatcode}{}{gregoriotex.tex}
-Macro to store the catcode for ``\@'' so that we can use said symbol in function names under Plain\TeX\ and then restore the original catcode after the package is done loading.
+Macro to store the catcode for ``@‘’ so that we can use said symbol in function names under Plain\TeX\ and then restore the original catcode after the package is done loading.
```

Comment:
After looking online, I found [this Wikibook](https://en.wikibooks.org/wiki/LaTeX/Plain_TeX) which use "Plain TeX" (i.e. "Plain" is capitalized, but their is a space between it and TeX).  I'm not sure if this should be considered an adequate authority, but have already made the switch to this.  I can go further (and make "plain" lowercase) if others think its a good idea.